locales.json

```diff
@@ -851,6 +851,26 @@
       "currentlyNoEvents": "Güncel bir etkinlik yok."
     }
   },
+  "news": {
+    "de": {
+      "local": "LOKALE NEWS",
+      "readMore": "Weiterlesen",
+      "currentlyNoNews": "Es gibt aktuell keine Nachrichten.",
+      "eNewsNo": "E-news No"
```

Comment:
Sollte nach der dt. rechschreibung `E-News No.` sein. Bin mir nicht sicher mit dem `.` am ende.

Reply:
Not sure what to do with it... Maybe we should just hardcode it or don't use it at all.

Reply:
I think we should just not use it at all and instead rather just display the id. It isn't used at all on the tunews homepage: 
![image](https://user-images.githubusercontent.com/29095487/82212959-b0ed6e00-9913-11ea-962b-5cb2cf6a6202.png)

Reply:
@maxammann what do you think?

Reply:
Discussed: We'll remove this